### ISSUE TITLE
Crashing 0.15.1 on app open/start

### ISSUE THREAD

**Issue by Marc-Gee** [PrimalHQ/primal-android-app#36]
## Describe the bug
Android app 0.15.1  keeps closing. Prior release worked great. A reboot fixes it once. Then the primal screen barely displays before closing. 
I can see it is on the latest feed, just before crashing. 

### Steps to reproduce
Opening primal 2nd and subsequent times after an Android reboot causes the app crash. Sorry, I don't know how to be more helpful. 

### Expected behavior
To not crash/close.

### Environment
 Pixel 6, android 13 
TQ3A.230805.001
Wifi.

**Comment by AleksandarIlic** [comment]
Thanks for reporting. Could you install the build with crash reporting tool so we could catch the crash log? If yes, please install primal-0.15.1-crash.apk from https://github.com/PrimalHQ/primal-android-app/releases/tag/0.15.1 and just crash the app. Thanks!

**Comment by d4f5409d** [comment]
Same here a13 1+ model 8

**Comment by s4msy** [comment]
Same crash, here are my logs:

```
type: crash
osVersion: google/sunfish/sunfish:13/TQ3A.230805.001/2023080800:user/release-keys
package: net.primal.android:32
process: net.primal.android
processUptime: 11073 + 68 ms
installer: com.android.packageinstaller

java.lang.IndexOutOfBoundsException: setSpan (-1 ... 47) starts before 0
	at android.text.SpannableStringInternal.checkRange(SpannableStringInternal.java:499)
	at android.text.SpannableStringInternal.setSpan(SpannableStringInternal.java:199)
	at android.text.SpannableStringInternal.setSpan(SpannableStringInternal.java:186)
	at android.text.SpannableString.setSpan(SpannableString.java:60)
	at v7.a.H(Unknown Source:12)
	at v7.a.E(Unknown Source:20)
	at x6.y.b2(Unknown Source:106)
	at androidx.compose.ui.platform.i0.o(Unknown Source:57)
	at androidx.compose.ui.platform.i0.s(Unknown Source:24)
	at androidx.compose.ui.platform.i0.K(Unknown Source:22)
	at androidx.compose.ui.platform.i0.J(Unknown Source:29)
	at androidx.compose.ui.platform.i0.K(Unknown Source:82)
	at androidx.compose.ui.platform.i0.J(Unknown Source:29)
	at androidx.compose.ui.platform.i0.K(Unknown Source:82)
	at androidx.compose.ui.platform.i0.J(Unknown Source:29)
	at androidx.compose.ui.platform.i0.K(Unknown Source:82)
	at androidx.compose.ui.platform.i0.K(Unknown Source:108)
	at androidx.compose.ui.platform.i0.J(Unknown Source:29)
	at androidx.compose.ui.platform.i0.l(Unknown Source:134)
	at androidx.compose.ui.platform.i0.A(Unknown Source:25)
	at androidx.activity.b.run(Unknown Source:293)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7940)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)

```

**Comment by Marc-Gee** [comment]
hi, any updates to fix this? (I assumed the crash-debug version would send you my logs). if not, please tell me how to extract them and i will upload them. thx. 
Looking forward to getting primal running again!

**Comment by d4f5409d** [comment]
> hi, any updates to fix this? (I assumed the crash-debug version would send you my logs). if not, please tell me how to extract them and i will upload them. thx. 
> Looking forward to getting primal running again!

I am sure they're working

**Comment by AleksandarIlic** [comment]
We were able to pinpoint the problem and working on the solution. The problem was in rendering hashtags where some devices were handling that gracefully while some other devices not at all (I assume due to difference in the underlying libraries). We will push the update as soon as the fix is done. Thank you for your patience.

**Comment by AleksandarIlic** [comment]
@d4f5409d @Marc-Gee Please reach out to me via DM on Nostr if you want to test the build with potential fix. I can not attach the APK here.

**Comment by d4f5409d** [comment]
send.vis.ee would be perfect tbh

**Comment by AleksandarIlic** [comment]
Fixed in 0.15.2.
